quarkchain/evm/specials.py

```diff
@@ -237,16 +237,32 @@ def proc_current_mnt_id(ext, msg):
 def proc_transfer_mnt(ext, msg):
     from quarkchain.evm.messages import apply_msg
 
-    # Data must be >= 96 bytes
-    if msg.data.size < 96:
-        return 0, 0, []
-    gascost = 3
-    if msg.gas < gascost:
+    # Data must be >= 96 bytes and static call not allowed
+    if msg.data.size < 96 or msg.static:
         return 0, 0, []
     to = utils.int_to_addr(msg.data.extract32(0))
     mnt = msg.data.extract32(32)
```

Comment:
Looks like the mnt can be greater than uint128.  Should we have any test to covert this?

Reply:
good catch. we have set `TOKEN_ID_MAX` to 4873763662273663091, and should check for all other methods `proc_transfer_mnt` and `proc_mint_mnt`

Reply:
done